.travis.yml

```diff
@@ -1,9 +1,11 @@
 language: node_js
 sudo: false
 node_js:
-  - 6.11.1
+  - 8.11.1
```

Comment:
Looks like we're using 8.11.3 in the `.node-version` -- can we use that here as well?

Reply:
Good catch!